eigrpd/eigrp_dump.h

```diff
@@ -138,9 +138,21 @@ extern unsigned long term_debug_eigrp_zebra;
 
 /* Prototypes. */
 extern const char *eigrp_if_name_string(struct eigrp_interface *);
-extern const char *eigrp_if_ip_string(struct eigrp_interface *);
-extern const char *eigrp_neigh_ip_string(struct eigrp_neighbor *);
-extern const char *eigrp_topology_ip_string(struct eigrp_prefix_entry *);
+static inline const char
+*eigrp_topology_ip_string(struct eigrp_prefix_entry *tn)
+{
+	return inet_ntoa(tn->destination->u.prefix4);
```

Comment:
oh man - hate to say this, but could we use `inet_ntop` instead, just to keep pushing that?

Reply:
wouldn't that matter if we actually had v6 support in eigrp?  There is so much work to be done here to get this to work that I'm not even sure it matters at this point in time( in other words we'll have to rewrite large swaths of code no matter what ).

Reply:
I guess it was just more about the use of the older, non-reentrant, etc. api.; but you're right, this is by far not the only place we've still got `ntoa` lurking...